client/package.json

```diff
@@ -17,7 +17,7 @@
     "build-storybook": "build-storybook -s ./packages/host/public ",
     "generate": "graphql-codegen --config codegen.yml",
     "android:run": "npx cap run android",
-    "android:build-remote_server": "cd ./packages/android && ./build_remote_server_libs.sh",
+    "android:build:server": "cd ./packages/android && ./build_remote_server_libs.sh",
```

Comment:
sorry. 
I usually forget whether it is `-` or `_`  and changing makes it a consistent pattern with `android:build:release` and `android:build:debug`